odoo/modules/module.py

```diff
@@ -117,20 +117,43 @@ def initialize_sys_path():
     """
     global hooked
 
+    # hook odoo.addons on data dir
     dd = os.path.normcase(tools.config.addons_data_dir)
     if os.access(dd, os.R_OK) and dd not in odoo.addons.__path__:
         odoo.addons.__path__.append(dd)
 
+    # hook odoo.addons on addons paths
     for ad in tools.config['addons_path'].split(','):
         ad = os.path.normcase(os.path.abspath(tools.ustr(ad.strip())))
         if ad not in odoo.addons.__path__:
             odoo.addons.__path__.append(ad)
 
-    # add base module path
+    # hook odoo.addons on community/addons/
```

Comment:
Please don't use this terminology `community/...`.
It is neither standard, nor correct (addons path includes private modules, too).
The original comment was clearer to me.